src/java/edu/slu/action/ObjectAction.java

```diff
@@ -727,6 +754,7 @@ public void getByProperties() throws IOException, ServletException, Exception{
      */
     public void saveNewObject() throws IOException, ServletException, Exception{
         if(null != processRequestBody(request, false) && methodApproval(request, "create")){
+            boolean intendedToHaveNewID = true;
```

Comment:
see below, but this flag doesn't make sense. If a user wants to add it, they just remove the property.

Reply:
agreed.